src/app/(contents)/post/PostList.tsx

```
@@ -0,0 +1,170 @@
+            <LineBasic />
+            <br />
+            <div></div>
+            <div className={styles.postListWrapper}>
+                {postList.length > 0 ? (
+                    postList.map((post, index) => {
+                        const diff = dayjs().diff(dayjs(post.createdAt), "hour")
+
+                        return (
+                            <Link
+                                key={post.id + "_" + post.createdAt + "_" + post.title}
+                                href={`/post/${post.id}`}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)